### ISSUE TITLE
Strange behavior of button in assessment approach[ED ]

### ISSUE THREAD

**Issue by Sylvie42** [w3c/wai-statements#69]
Hello,
At [generate an accessibility statement](https://w3c.github.io/wai-statements/planning/statements/generator/#create)
Location: Assessment approach
Issue: There is a button "show info" + that cannot be expanded although announced as collapsed by NVDA.
May be a coding error? 
Editor's discretion to see what is going on there.

**Comment by nitedog** [comment]
Bug fixed
